app/src/main/java/com/framgia/englishconversation/data/model/TimelineModel.java

```diff
@@ -227,6 +238,20 @@ public int getViewType() {
         return getMedias() != null ? getMedias().size() : 0;
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == null) {
+            return false;
+        }
+        if (obj instanceof TimelineModel) {
+            TimelineModel other = (TimelineModel) obj;
+            return other.getId().equals(getId());
+        }
```

Comment:
ERROR: '}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).